cdap-distributions/src/packer/scripts/cookbook-setup.sh

```diff
@@ -20,8 +20,10 @@
 
 # Grab cookbooks using knife
 for cb in hadoop idea maven nodejs cdap ; do
-  knife cookbook site install $cb
+  knife cookbook site install $cb || echo "Cannot fetch cookbook $cb" && exit 1
```

Comment:
How did I miss these? ;-)
precedence order for both bitwise and logical tests is always NOT, AND, OR